mount/mount_windows.go

```diff
@@ -46,22 +72,42 @@ func (m *Mount) mount(target string) error {
 	if err = hcsshim.ActivateLayer(di, layerID); err != nil {
 		return fmt.Errorf("failed to activate layer %s: %w", m.Source, err)
 	}
+	defer func() {
+		if err != nil {
+			hcsshim.DeactivateLayer(di, layerID)
```

Comment:
This and Unprepare always scare me in terms of what they could fail with, so wanna grab the error and log here and same for on line 86? Up to you, it would bring me peace of mind 😭